meson_options.txt

```diff
@@ -0,0 +1,22 @@
+option('yaml',
+  type : 'feature',
+  value: 'enabled',
+  description : 'Whether to support YAML manifests',
+)
+option('dwarf_header_dir',
+  type : 'string',
+  value : '',
+  description : 'The directory where the dwarf.h header is located, if not in include directories.',
+)
+option('documentation',
```

Comment:
Please use whatever the corresponding option is called in GNOME/GLib projects, to maximize the ability to share an option value between projects. If this is controlling the gtk-doc-generated reference documentation, I think that's usually `gtk_doc`?

Reply:
But it's not gtk-doc, it's docbook/xmlto stuff. But anyway it seems like this option is duplicated with docbook-docs, since I just copied the available autoconf options but it only enabled checks for DTD catalogs and XSL stylesheets, which should be done when enabling the main docbook docs option. So I'll just drop it.